tox.ini

```diff
@@ -59,10 +59,11 @@ commands =
         --skip {tox_root}/lib \
         --skip {tox_root}/venv \
         --skip {tox_root}/.mypy_cache \
-        --skip {tox_root}/icon.svg
+        --skip {tox_root}/icon.svg \
+        --skip {tox_root}/poetry.lock
     poetry run codespell {[vars]lib_path}
 
-    poetry run ruff {[vars]all_path}
+    poetry run ruff {[vars]all_path} --exclude tests/integration/app-charm/lib/
```

Comment:
Excluding tester lib because for some reason it [fails on GH](https://github.com/canonical/kafka-operator/actions/runs/4427320637/jobs/7764741828) (passes locally):
```
 lint: commands[3]> poetry run ruff /home/runner/work/kafka-operator/kafka-operator/src /home/runner/work/kafka-operator/kafka-operator/tests
tests/integration/app-charm/lib/charms/operator_libs_linux/v0/apt.py:724:5: D417 Missing argument description in the docstring: `package_names`
tests/integration/app-charm/lib/charms/operator_libs_linux/v1/snap.py:816:5: D417 Missing argument description in the docstring: `cohort`
tests/integration/app-charm/lib/charms/operator_libs_linux/v1/snap.py:864:5: D417 Missing argument descriptions in the docstring: `cohort`, `snap_names`
Found 3 errors.
```

Reply:
If you don't mind, try removing this? I *think* I fixed this in a recent merge, if not we can add back.